scaffolding-node/lib/scaffolding.sh

```diff
@@ -134,6 +134,11 @@ scaffolding_modules_install() {
         cp -av tsconfig.json "$CACHE_PATH/"
       fi
 
+      # Important for tsoa
+      if [[ ! -f "$CACHE_PATH/tsoa.json" ]]; then
```

Comment:
I'm taking a look at the logic of this and have a question.

Looks like we check whether there is NOT a $CACHE_PATH/tsoa.json file and, should that be true, copy the tsoa.json file into the $CACHE_PATH.  I'm concerned that this will cause problems with node applications that do not use tsoa and therefore would not have a tsoa.json file at all.

What if we did this - test whether there is a tsoa.json file in the current directory and, if there is, copy it to $CACHE_PATH?

i.e. 

```
    if [[ -f "tsoa.json" ]]; then
        cp -av tsoa.json "$CACHE_PATH/"
    fi
```

Reply:
Oops! You're absolutely right. That makes a lot more sense. I'll update.